### ISSUE TITLE
Remove spacing between bullet points and lists

### ISSUE THREAD

**Issue by LeMurphant** [StampyAI/stampy-ui#613]
![spacing](https://github.com/StampyAI/stampy-ui/assets/16197732/9890636b-07ce-41ae-b0a5-65423cdf285c)
https://docs.google.com/document/d/1YKA4i_3rfMMGrWLtvzNy9IplYgS4Ialdc6f-Rfu-CBg/edit
https://aisafety.info/questions/9B85/Isn't-the-real-concern-misuse

It seems to me (and Taylor and Steven) that the vertical spacing between bullet points (or numbered points) is excessive

**Comment by bryceerobertson** [comment]
@LeMurphant is this looking better now?

**Comment by LeMurphant** [comment]
Unfortunately we now have issues with insufficient spacing in some cases
![spacing](https://github.com/user-attachments/assets/e196c16b-2d4d-497f-bba7-17588c8ab89a)
![spacing_gdocs](https://github.com/user-attachments/assets/cac980f3-6485-4769-9fd4-12748dd104a9)

**Comment by LeMurphant** [comment]
This might be related to https://github.com/StampyAI/stampy-ui/issues/352

**Comment by Algon-33** [comment]
Claire, @LeMurphant's friend, wants there to be more space after embedded videos in our articles. This seems like a related problem. Please fix it.

**Comment by LeMurphant** [comment]
There are also issues with increasing the space after sub-headings, e.g.
https://aisafety.info/questions/NM3E/13:-We-are-not-reliably-on-track-to-solving-these-problems

**Comment by Algon-33** [comment]
This is relatively high priority because of how it makes the site feel.

**Comment by LeMurphant** [comment]
@Algon-33 I agree. IIRC I diagnosed that the issue was in GDocsRelatedThings, I'll try to get to it before next Monday

**Comment by Algon-33** [comment]
TY

**Comment by LeMurphant** [comment]
The fix did not work, the rich text in Coda seems unchanged.

My guess is that the issue is that the parser proceeds differently than devtools.js, I will investigate

**Comment by Algon-33** [comment]
Ah, that sucks.
